src/lycophron/cli.py

```diff
@@ -92,6 +93,218 @@ def configure():
     pass
 
 
+@lycophron.command()
+@click.option("--filename", prompt="Filename", default="data.xlsx")
```

Comment:
comment: this could also be an optional argument, using the [File argument type](https://click.palletsprojects.com/en/8.1.x/arguments/#file-arguments) from Click.